.travis.yml

```diff
@@ -1,32 +1,22 @@
 sudo: required
 dist: trusty
 services:
-  - rabbitmq
-  - mongodb
-env:
-  - CXX=g++-4.8 TRAVIS_NODE_VERSION="6.9.4"
+- rabbitmq
+- mongodb
 language: python
 python:
-  - "3.5"
-  - "3.5-dev"
-  - "nightly"
+- 3.5
+- 3.6
 install:
-  - pip install -r requirements.txt
-  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
```

Comment:
The default node.js version on travis should be 7.4, per https://github.com/travis-ci/travis-ci/issues/5218 and build log.